launcher/ui/MainWindow.cpp

```diff
@@ -1137,6 +1144,10 @@ void MainWindow::showInstanceContextMenu(const QPoint &pos)
             connect(actionDeleteGroup, SIGNAL(triggered(bool)), SLOT(deleteGroup()));
             actions.append(actionDeleteGroup);
         }
+
+        QAction *actionUndoTrashInstance = new QAction("Undo last trash instance", this);
+        connect(actionUndoTrashInstance, SIGNAL(triggered(bool)), SLOT(undoTrashInstance()));
+        actions.append(actionUndoTrashInstance);
```

Comment:
Minor nitpick, but maybe this action could only be enabled if there's actually something to undo?

Reply:
@ryanccn this is not resolved (:

My computer doesn't have a trash, still the action is enabled when the menu bar exists. So, having only the toolbar makes the action disabled, as it should be, but otherwise it's enabled.

Reply:
(GitHub is being an idiot, sorry if you receive a lot of notifications)
This is not resolved! :o

You see, I don't have a trash in my system. So, one would assume this action would be disabled at all times. But, it is in fact enabled when the menu bar is present (at all times). When there's only the toolbar, it is disabled as normal.

Reply:
@ryanccn idk if you saw this, GH was acting weird that night so maybe it didn't notify you :p